mailet/standard/src/test/java/org/apache/james/transport/mailets/ContactExtractorTest.java

```diff
@@ -162,6 +162,26 @@ public void serviceShouldUseFromFieldWhenNoSender() throws Exception {
     }
 
     @Test
+    public void serviceShouldOnlyPickSenderAddress() throws Exception {
+        MimeMessage message = MimeMessageBuilder.mimeMessageBuilder()
+            .setSender("Sender <" + SENDER + ">")
+            .addToRecipient("To <" + TO + ">")
+            .setSubject("Contact collection Rocks")
+            .setText("This is my email")
+            .build();
+        FakeMail mail = FakeMail.builder().mimeMessage(message)
+            .sender(new MailAddress(SENDER))
+            .recipient(new MailAddress(TO))
+            .build();
+        mailet.init(mailetConfig);
+
+        String expectedMessage = "{\"userEmail\" : \"" + SENDER + "\", \"emails\" : [ \"To <" + TO + ">\" ]}";
+        mailet.service(mail);
+
+        assertThatJson(mail.getAttribute(ATTRIBUTE).toString()).isEqualTo(expectedMessage);
+    }
```

Comment:
are you able to add a test with a group address type?

Reply:
I don't know the damn syntax